packages/validator/src/cli-validator/run-validator.js

```diff
@@ -246,9 +249,14 @@ const processInput = async function(program) {
         chalk
       );
 
-      process.chdir(path.dirname(validFile));
-      // let spectral handle the parsing of the original swagger/oa3 document
-      spectralResults = await spectral.run(originalFile);
+      const fileExtension = ext.getFileExtension(validFile);
+      let parser = Parsers.Json;
+      if (['yaml', 'yml'].includes(fileExtension)) {
+        parser = Parsers.Yaml;
+      }
+
+      const doc = new Document(originalFile, parser, validFile);
```

Comment:
This is the meat of the change - using a Spectral `Document` is required for `overrides`